lib/plugins.js

```diff
@@ -817,74 +812,74 @@ function plugins (settings, deviceId) {
       }
       : null
     
-    const stacker1fAlert = device.numberOfStackers >= 1 && isUnitLow(device.cashUnits.stacker1f, getCashUnitCapacity(device.model, 'stacker'), notifications.fillingPercentageStacker1f)
+    const recycler1Alert = device.numberOfRecyclers >= 1 && isUnitLow(device.cashUnits.recycler1, getCashUnitCapacity(device.model, 'recycler'), notifications.fillingPercentageRecycler1)
       ? {
         code: 'LOW_RECYCLER_STACKER',
         cassette: 4,
```

Comment:
what are these `cassette: 4` fields?

Reply:
I'll look into it further on the notification specific card.